docs/how-to/use-trained-models.txt

```diff
@@ -14,15 +14,16 @@ This guide discusses:
 #. Storing additional user-defined metadata in a checkpoint.
 #. Using the Determined CLI to download checkpoints to disk.
 
+For specific on the checkpoint API, see
```

Comment:
Reword?  Maybe something like:

"The Checkpoint Export API is a subset of the features found in ..."